src/rdkafka_cgrp.c

```diff
@@ -1400,28 +1921,29 @@ static void rd_kafka_cgrp_join (rd_kafka_cgrp_t *rkcg) {
 /**
  * Rejoin group on update to effective subscribed topics list
  */
-static void rd_kafka_cgrp_rejoin (rd_kafka_cgrp_t *rkcg) {
+static void rd_kafka_cgrp_revoke_rejoin (rd_kafka_cgrp_t *rkcg) {
         /*
```

Comment:
i'm not actually sure what to use as 'state is rejoin' (what precisely is this). all I was doing here is making the previous wording a bit more precise, i'm not going to go further and make the suggested change because it's a minor thing and i don't want to put in the effort right now, given it's saturday and stuff.